cdap-proto/src/main/java/io/cdap/cdap/proto/security/InstancePermission.java

```diff
@@ -24,6 +24,10 @@
    * Permission to tether other clusters with a CDAP instance
    */
   TETHER,
+  /**
+   * Permission to gather health check information from kubernetes pods
```

Comment:
nit: JVM instead of kubernetes pods, as the implementation is not environment-aware